### DISCUSSION TITLE
How to renew the project CVE scan?

### DISCUSSION THREAD

**Discussion by JN-CSIRT** [DependencyTrack/dependency-track#2191] [Q&A]
Hello,
after create new project version with components & CVE migrations, after update the components versions some CVEs are not found or CVE notifications are not sent.
Is there an option/command to re-scan the new project version components for CVEs and resend the Notification for identified CVEs?

Thanks a lot!

**Comment by syalioune** [comment]
See https://github.com/DependencyTrack/dependency-track/issues/2116. There will be a new button to do exactly that in 4.7 release.
In the meantime, you can export and reimport the SBOM, it will launch an analysis of the project.

**Comment by JN-CSIRT** [comment]
Additional point is that after migration and change Component version/CPE, old CVEs for old version/CPE still be available in Vulnerabilities list.
Only after component is deleted and created again, correct CVE only are listed...
Can you please confirm/check?

**Reply by syalioune** [reply]
Yes, you're right : updating an existing component in a project won't delete the old vulnerabilities (I think that it should). Only a delete/create will.
I suggest you create a distinct issue for this.

**Comment by JN-CSIRT** [comment]
An Issue is created: #2284

**Comment by JN-CSIRT** [comment]
Hello DT Team,
I've create an Issue but there is no progress! Can you please check it?
